ciao-launcher/instance_test.go

```diff
@@ -209,18 +209,57 @@ func (v *instanceTestState) verifyStatsUpdate(t *testing.T, cmd interface{}) {
 	}
 }
 
-func (v *instanceTestState) expectStatsUpdate(t *testing.T, ovsCh <-chan interface{}) bool {
+func (v *instanceTestState) getStatsUpdate(t *testing.T, ovsCh <-chan interface{}) *ovsStatsUpdateCmd {
 	var cmd interface{}
 	select {
 	case cmd = <-ovsCh:
 	case <-time.After(time.Second):
 		t.Error("Timed out waiting for ovsStatsUpdateCmd")
-		return false
+		return nil
 	}
 	stats, ok := cmd.(*ovsStatsUpdateCmd)
 	if !ok {
 		t.Error("Unexpected Command received on ovsCh")
 	}
+	return stats
+}
+
+func (v *instanceTestState) expectStatsUpdateWithVolumes(t *testing.T,
+	ovsCh <-chan interface{}, volumes []string) bool {
```

Comment:
Is there an intended usage for the bool return coming?  Looks like all callers do "_ := expectStatsUpdateWithVolumes()".  I'm not sure the code would be notably more succinct with an error return only and no  *testing.T input, but it feels extraneous as is.